plugins/amazonq/chat/jetbrains-community/src/software/aws/toolkits/jetbrains/services/amazonqFeatureDev/controller/FeatureDevController.kt

```diff
@@ -284,6 +288,26 @@ class FeatureDevController(
         }
     }
 
+    private suspend fun handleStopMessage(message: IncomingFeatureDevMessage.StopResponse) {
+        var session: Session? = null
```

Comment:
```suggestion
```
declared too early.
var can be val

Reply:
Value is reassigned on line 305, so need to be var. I've removed the redundant null